api-generation/src/main/scala/ai/h2o/sparkling/api/generation/r/ConfigurationTemplate.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.api.generation.r
+
+import java.lang.reflect.Method
+
+object ConfigurationTemplate extends ((Array[Method], Array[Method], Class[_]) => String) {
+
+  def apply(getters: Array[Method], setters: Array[Method], entity: Class[_]): String = {
+
+    s"""#
+       |#
+       |# Licensed to the Apache Software Foundation (ASF) under one or more
+       |# contributor license agreements.  See the NOTICE file distributed with
+       |# this work for additional information regarding copyright ownership.
+       |# The ASF licenses this file to You under the Apache License, Version 2.0
+       |# (the "License"); you may not use this file except in compliance with
+       |# the License.  You may obtain a copy of the License at
+       |#
+       |#    http://www.apache.org/licenses/LICENSE-2.0
+       |#
+       |# Unless required by applicable law or agreed to in writing, software
+       |# distributed under the License is distributed on an "AS IS" BASIS,
+       |# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       |# See the License for the specific language governing permissions and
+       |# limitations under the License.
+       |#
+       |
+       |if (!exists("ConfUtils.getOption", mode = "function")) source(file.path("R", "ConfUtils.R"))
+       |
+       |#' @export ExternalBackendConf
```

Comment:
is this line needed?

Reply:
Maybe it should be  constant?

Reply:
Ah, my bad. This line is required but needs to be parametrized, thanks!